src/environment/Permittivity.cpp

```diff
@@ -30,81 +30,22 @@
 namespace mrchem {
 
 Permittivity::Permittivity(const mrchem::Cavity cavity, double epsilon_in, double epsilon_out, std::string formulation)
-        : epsilon_in(epsilon_in)
-        , epsilon_out(epsilon_out)
-        , formulation(formulation)
-        , cavity(cavity) {}
+        : StepFunction(cavity, epsilon_in, epsilon_out) {}
 
 double Permittivity::evalf(const mrcpp::Coord<3> &r) const {
-    auto epsilon = epsilon_in * std::exp(std::log(epsilon_out / epsilon_in) * (1 - this->cavity.evalf(r)));
+    auto epsilon = this->in * std::exp(std::log(this->out / this->in) * (1 - this->cavity.evalf(r)));
     if (inverse) {
         return 1 / epsilon;
     } else {
         return epsilon;
     }
```

Comment:
I like ternary `if`s, but I leave it up to you to decide what's preferable here?
```suggestion
    return (this->inverse) : 1.0 / epsilon ? epsilon; 
```